server/graphql/v1/CollectiveInterface.js

```diff
@@ -870,6 +877,19 @@ export const CollectiveInterfaceType = new GraphQLInterfaceType({
   },
 });
 
+const FeaturesFields = () => {
+  return FeaturesList.reduce(
+    (obj, feature) =>
+      Object.assign(obj, {
+        [feature]: {
+          type: CollectiveFeatureStatus,
+          resolve: getFeatureStatusResolver(`${feature}`),
```

Comment:
Why not directly:

```suggestion
          resolve: getFeatureStatusResolver(feature),
```

?